index.bs

```diff
@@ -3409,6 +3434,13 @@ Axel Nennker, Kimberly Paulhamus, Adam Powers, Yaron Sheffer, Mike West, Jeffrey
     "status": "FIDO Alliance Implementation Draft"
   },
 
+  "FIDO-Allowed-Restricted-Operating-Environments": {
+    "authors": ["L. Lundblade", "M. Karlsson"],
+    "title": "FIDO Authenticator Allowed Restricted Operating Environments List",
+    "href": "https://fidoalliance.org/wp-content/uploads/fido-authenticator-allowed-restricted-operating-environments-list-2.pdf",
+    "status": "FIDO Alliance Final Document"
```

Comment:
The link you have doesn't look like a final document: is it really an editor's draft until it's published to https://fidoalliance.org/specs?